author/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from . import views
+
+app_name = "author"
+urlpatterns = [
+    path('<int:pk>',views.BloggerDetailView.as_view(), name='auth-detail')
```

Comment:
You forgot to add `/`. Change name to `author-detail`, as auth and author are different things.